### PR TITLE
Release with GH Action

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -141,3 +141,43 @@ jobs:
           # Make sure the lint command can run without errors.
           # TODO: run the full suite of E2E tests on Windows.
           tmp/windows/kube-linter.exe lint "tests/checks/access-to-create-pods.yml"
+
+  update_release_draft:
+    if: github.ref == 'refs/heads/main'
```
Comment (viswajithiii) [inline .github/workflows/build.yaml:146]: Shouldn't this only run on tags?

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    label: "chore"
+  - title: '⬆️ Dependencies'
+    collapse-after: 5
+    labels:
+      - 'dependencies'
+change-template: "- $TITLE @$AUTHOR (#$NUMBER)"
+template: |
+  ## Changes
+
+  $CHANGES
```
Comment (viswajithiii) [inline .github/release-drafter.yml:23]: Add newline?

**Review by viswajithiii — COMMENTED** [review]
Shouldn't we simultaneously delete the CircleCI release job?

File: .github/release-drafter.yml (hunk 2 above)
Comment (janisz) [reply .github/release-drafter.yml]: Fixed

**Review by dhaus67 — APPROVED** [review]
LGTM modulo removing the existing circle ci job

**Pull request merged**